script/worker

```diff
@@ -141,6 +140,7 @@ sub read_worker_config {
     my $host_settings;
     $host ||= $sets->{'HOST'} ||= 'localhost';
     delete $sets->{'HOST'};
+    die "Hosts in workers.ini is malformed, use space as separators ($host)" if grep /,/, $host;
```

Comment:
Hmm.. I understand the reasoning, but (1) comma is valid part of query part or URI (I can imagine having openQA host as http://example.domain/openqa,1 ) and (2) /ignoring (1)/ as you already detect it we may as well allow that as a delimiter. IMO it's useless to disallow presence in host string and not use it as separator at the same time.